scripts/postbuild.ts

```diff
@@ -0,0 +1,36 @@
+import { readFileSync, writeFileSync } from 'node:fs'
+import { resolve } from 'node:path'
+
+// fix cjs export default
+const cjsModule = resolve('dist/module.cjs')
+const cjsModuleContent = readFileSync(cjsModule, 'utf-8')
+writeFileSync(
+  cjsModule,
+  cjsModuleContent.replace(
+    'module.exports = function',
+    'exports.default = function',
```

Comment:
What did this address?

Reply:
Node10

Reply:
https://arethetypeswrong.github.io/?p=%40vite-pwa%2Fnuxt%400.3.2

Reply:
removed from postbuild script